xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2386,6 +2388,14 @@ void CVideoPlayer::OnExit()
 {
   CLog::Log(LOGNOTICE, "CVideoPlayer::OnExit()");
 
+  bool hdr_capable, hdr_enabled;
+  DX::DeviceResources::Get()->DetectDisplayHDRcapable(hdr_capable, hdr_enabled);
+
+  if (hdr_enabled && DX::Windowing()->Is_10bitSwapchain())
+  {
+    DX::DeviceResources::Get()->Clear_HDR_MetaData();
```

Comment:
This isn't going to work. You're putting windows specific code into platform agnostic calls.

Reply:
Sorry, I don't know the Kodi code structure well. It needs fixing. But just move the code to the right place.

Reply:
this must be moved to WinRenderer.cpp

Reply:
OK